stacker/providers/aws/default.py

```diff
@@ -914,6 +947,8 @@ def noninteractive_changeset_update(self, fqn, template, old_parameters,
             'UPDATE', service_role=self.service_role, **kwargs
         )
 
+        self.deal_with_changeset_stack_policy(fqn, stack_policy)
```

Comment:
Ok cool, so this is the bug, you refactored the logic so you could call it here.